src/main/java/Duke.java

```diff
@@ -1,18 +1,27 @@
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
 import duke.Command;
 import duke.Parser;
 import duke.Storage;
 import duke.Task;
 import duke.TaskList;
 import duke.Ui;
-import java.util.Scanner;
-import java.nio.file.Path;
-import java.nio.file.Paths;
 
+
+/**
+ * Main class Duke that runs the application.
+ */
 public class Duke {
```

Comment:
Very enlightening to see such a consistent level of abstraction within Duke class.